src/nvim/buffer.c

```diff
@@ -1950,15 +1950,8 @@ void free_buf_options(buf_T *buf, int free_p_ff)
     xfree(buf->b_p_vsts_nopaste);
   }
   buf->b_p_vsts_nopaste = NULL;
-  if (buf->b_p_vsts_array) {
-    xfree(buf->b_p_vsts_array);
-  }
-  buf->b_p_vsts_array = NULL;
```

Comment:
Ah that's not right, it shouldn't have been deleted. Will force-push without this later today.

Reply:
Done. @janlazo